tasks/functions/functions.js

```diff
@@ -51,29 +79,48 @@ function getGoogle(n) {
  *    age: 42
  * }
  */
-function getUser(firstName, lastName, age) {
-
-}
+/*function getUser(firstName = null, lastName = null, age = null) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+  }
+  
+  var person = new getUser("John", "Doe", 42);
+  console.log (person);
+  var person = new getUser("John", "Doe");
+  console.log (person);
+*/
+function getUser(firstName = null, lastName = null, age = null) {
+  var obj = {
+    firstName: firstName,
+    lastName: lastName,
+    age: age
+};
+  return obj;
+};
+getUser("John", "Dou", 42);
 
 /**
  * write function that calculates total path traveled.
  * path represended as array of objects with field distance and direction
  * e.g [{direction: "Kiyv - Minsk", distance: 567}, {direction: "Kiyv - Paris", distance: 2402}]
  */
 
-function getTotalPath(path) {
-
+const path = [
+    {  direction: "Kiyv - Minsk", distance: 567},
+    {  direction: "Kiyv - Paris", distance: 2402}
+];
+function getTotalPath(arr){
+      let total = 0;
+      let i;
+       for (i = 0; i < arr.length; i++) {
```

Comment:
it's better to define the iterable variable inside the for loop:
for (let i = 0; i < arr.length; i++)...

Reply:
done